plotly/graph_reference.py

```diff
@@ -410,12 +410,25 @@ def _patch_objects():
                          'attribute_paths': layout_attribute_paths,
                          'additional_attributes': {}}
 
-    figure_attributes = {'layout': {'role': 'object'},
-                         'data': {'role': 'object', '_isLinkedToArray': True}}
+    figure_attributes = {
+        'layout': {'role': 'object'},
+        'data': {'role': 'object', '_isLinkedToArray': True},
+        'frames': {'role': 'object', '_isLinkedToArray': True}
+    }
     OBJECTS['figure'] = {'meta_paths': [],
                          'attribute_paths': [],
                          'additional_attributes': figure_attributes}
 
+    frame_entry_attributes = {
+        'data': {'role': 'object', '_isLinkedToArray': True},
+        'group': {'role': 'info'},
+        'layout': {'role': 'object'},
+        'name': {'role': 'info'}
+    }
```

Comment:
Unfortunately there's not a dedicated doc page since this doesn't currently live on the plot scheme. In lieu of that, the attributes are defined [here](https://github.com/plotly/plotly.js/blob/master/src/plots/frame_attributes.js). @etpinard is there some way I can add these to the plot schema? The way animation config and frame attributes live outside of the rest of plotly is driving me a little crazy.